modules/dns/README.md

```diff
@@ -31,6 +31,33 @@ The returned `flags` parameter means that when you run a `terraform plan` comman
             # (12 unchanged attributes hidden)
         }
 
+The DNS records also can be imported using a file in IBM Cloud Internet Services. This module allows the import of DNS records in two ways:
+- Directly importing the `dns_record.txt` in BIND format.
+- Do base64 encoding on the `dns_record.txt` in BIND format and pass it as a string.
+
+For converting the dns_record text file to Base64 encoding.
+```sh
+cat dns_records.txt | base64
+```
+
+When the `dns_records` are successfully added through file import by any of the above methods then `records_added` and `total_records_parsed` is not set in the configuration.
```

Comment:
When the `dns_records` are successfully imported then `records_added` and `total_records_parsed` get set in the configuration.

Reply:
Made the changes.